PaxLabs/mynewt-core
# PKF-432: Modify sensor driver to allow sensor framework to change sam…

…ple rate

Test:
1. turn on the device. Don't insert a pod. Disable uart by "power control off uart0"
2. Check the current right after the command.
2. wait 10 seconds for system to enter sleep mode and remain still
3. Check current again. The current should go down.
